Examples/rc-maestro/rc-maestro/Tests/RcMaestroTests.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+import XCTest
```

Comment:
Why not Swift Testing 😬 ?

Reply:
We need to bump Xcode in the CI first 🚀 

Also, two new tools in one PR is already enough 😆